.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -25,61 +25,79 @@ jobs:
     steps:
       - name: "Checkout latest commit"
         uses: actions/checkout@v2
-      - name: "Set up Python 3.8"
+      - name: "Set up Python"
         uses: actions/setup-python@v2
         with:
+          # 3.8 is chosen because it seems to be the fastest for <3.9
+          # (3.9 excluded because it seems to be still very unstable)
           python-version: 3.8
       - name: "Install dependencies"
+        # language=bash
         run: |
-          python -m pip install --upgrade pip
-          pip install tox
-      - name: "Execute tox -e qa,docs"
+          python -m pip install --upgrade pip setuptools wheel
+          pip install colorama pytype flake8 pytest pytest-mock sphinx pickle5 check-manifest
+          python setup.py develop
+      - name: "flake8 Style Checking"
+        # language=bash
         run: |
-          tox -e qa,docs
```

Comment:
"Unfolding" tox into separate, direct invocation of test stuff should make GA checks faster as the runner no longer has to first create virtualenvs.

The drawback is of course if the steps within tox.ini changes, we have to edit the GA steps as well, but I don't think we'll be changing them any time soon after this heavylift transition to pure pytest.